### PR TITLE
Implemented OOP concepts

### PR DESCRIPTION
**Pull request by ARJUN-R34**
Implemented method abstraction and encapsulation.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,60 +1,85 @@
 const cryptojs = require('crypto-js');
 
-const { getRequest, postRequest, generateToken, sendTransaction } = require('./utils/helper')
+const { getRequest, postRequest, getAccessToken, sendTransaction } = require('./utils/helper')
 const { AUTH_SERVICE_URL } = require('./config')
 
-class PBTS {
-    constructor(authToken) {
-        this.authToken = authToken;
+async function AccessToken({ handlename, password, authToken }) {
```
Comment (apoorvq) [inline index.js:6]: AccessToken is not camel case.

Comment (apoorvq) [inline index.js:6]: getAccessToken

File: index.js [hunk 2]
```diff
@@ -1,60 +1,85 @@
...
         }
-        return "There has been an issue. Pleaser try again later."
+        return { error: "There has been an issue. Pleaser try again later." }
     }
 
-    async decryptAndSignTransaction(payload) {
-        const { password, handlename, rawTx, infuraKey, rpcUrl } = payload;
-
-        const { response: accessToken, error } = await generateToken({ handlename, password, authToken: this.authToken });
+    async storeKey({ privateKey, password, handlename }) {
```
Comment (apoorvq) [inline index.js:56]: What is this for?

File: index.js (hunk 1 above)
Comment (ARJUN-R34) [reply index.js]: fixed it.

File: index.js (hunk 2 above)
Comment (ARJUN-R34) [reply index.js]: storeKey() function encrypts the private key and stores it in the db.

**Comment by apoorvq** [comment]
Please resolve conflicts. @ARJUN-R34

**Comment by ARJUN-R34** [comment]
> Please resolve conflicts. @ARJUN-R34

Conflicts resolved and branch ready to merge.

**Pull request merged**